### ISSUE TITLE
scale admin dash to fit on screen

### ISSUE THREAD

**Issue by jessyurko** [berkmancenter/bookanook#154]
can we scale the admin tables (nooks, reservations, welcome) to fit within screen bounds?

**Comment by shubhpatel108** [comment]
@jessyurko can we stop showing 'created_at' on reservations index table?
I have updated how timestamps are shown but the table doesn't fit in.

**Comment by jessyurko** [comment]
Yes, it doesn't need to be on the index table; if it's easy to add it to the edit reservation view (just so it's visible somewhere) that would be useful, but not a priority.
